board/main.c

```diff
@@ -482,6 +512,12 @@ int usb_cb_control_msg(USB_Setup_TypeDef *setup, uint8_t *resp, bool hardwired)
         }
         break;
       }
+    // **** 0xf3: Set harness relay to (not) intercept (black panda only).
+    case 0xf3:
```

Comment:
Do we need this? Why can't we just use the safety mode?

Reply:
currently, the safety mode persists until panda is disconnected from EON. We'll have to refactor things a bit.

Reply:
Removed this part and set the intercept to be on when not in NOOUTPUT or ELM mode.